fuzz/cycle.sh

```diff
@@ -7,7 +7,8 @@
 
 set -e
 REPO_DIR=$(git rev-parse --show-toplevel)
-# shellcheck source=./fuzz-util.sh
+# can't find the file because of the ENV var
+# shellcheck source=/dev/null
```

Comment:
Why do you silence this like this man instead of? 
```
# shellcheck disable=SC1090
```

(I hit the same warning working on https://github.com/rust-bitcoin/rust-secp256k1/pull/699.)

Reply:
It was the same approach I did in the original `rust-bitcoin` PR and this is the way to adopt a Exception per the shellcheck wiki: https://www.shellcheck.net/wiki/SC1090

Reply:
No worries, cheers.